Resources/template.json

```diff
@@ -0,0 +1,222 @@
+                  "value": "diversitydetails"
+                },
+                {
+                  "name": "DiversityLeaseCollectionName",
+                  "value": "diversity-lease"
+                },
+                {
+                  "name": "DiversityLeaseCollectionPrefix",
+                  "value": "DiversityChangeFeedPrefix"
+                },
+                {
+                  "name": "InteractionCollectionId",
+                  "value": "interactions"
+                },
+                {
+                  "name": "InteractionDatabaseId",
+                  "value": "interactions"
+                },
+                {
+                  "name": "InteractionLeaseCollectionName",
+                  "value": "interaction-lease"
+                },
+                {
+                  "name": "InteractionLeaseCollectionPrefix",
+                  "value": "InteractionChangeFeedPrefix"
+                },
+                {
+                  "name": "SubscriptionCollectionId",
+                  "value": "subscriptions"
+                },
+                {
+                  "name": "SubscriptionDatabaseId",
+                  "value": "subscriptions"
+                },
+                {
+                  "name": "SubscriptionLeaseCollectionName",
+                  "value": "subscription-lease"
+                },
+                {
+                  "name": "SubscriptionLeaseCollectionPrefix",
+                  "value": "SubscriptionChangeFeedPrefix"
+                },
+                {
+                  "name": "TransferCollectionId",
+                  "value": "transfers"
+                },
+                {
+                  "name": "TransferDatabaseId",
+                  "value": "transfers"
+                },
+                {
+                  "name": "TransferLeaseCollectionName",
+                  "value": "transfer-lease"
+                },
+                {
+                  "name": "TransferLeaseCollectionPrefix",
+                  "value": "TransferChangeFeedPrefix"
+                },
+                {
+                  "name": "WebChatCollectionId",
+                  "value": "webchats"
+                },
+                {
+                  "name": "WebChatDatabaseId",
+                  "value": "webchats"
+                },
+                {
+                  "name": "WebChatLeaseCollectionName",
+                  "value": "WebChat-lease"
+                },
+                {
+                  "name": "WebChatLeaseCollectionPrefix",
+                  "value": "WebChatChangeFeedPrefix"
+                },
+                {
+                  "name": "ChangeFeedQueueName",
+                  "value": "dss.changefeedqueue"
+                },
+                {
+                  "name": "ServiceBusConnectionString",
+                  "value": ""
+                },
+                {
+                  "name": "ConnectionString",
```

Comment:
Bad app setting name, see if K can change this to make it clear what the ConnectionString is for.